Domain/basicModelInterfaces.go

```diff
@@ -0,0 +1,20 @@
+package Domain
+
+type UserModelInterface interface {
+	GetFirstName() string
+	GetLastName() string
+	GetBirthDate() string
```

Comment:
In my opinion in domain model bidthDate should be type of date, sex - enum(or int, that could be stored as const in userModel class), material status - same as sex.

Reply:
@malinink actually, UserModelInterface isn't a Domain Mode

Reply:
It is not clear to me